misc/images/materialized/README.md

```diff
@@ -1,19 +1,18 @@
-This is an all-in-one Docker image of Materialize, a fast, distributed SQL
-database built on streaming internals. This image is used for **internal**
-development and testing of Materialize.
-
-> **Warning**
->
-> You **should not** run production deployments using this Docker image.
->
-> This Docker image is **not** supported by Materialize.
->
-> This Docker image does **not** support version upgrades.
->
-> The performance characteristics of this Docker image are **not** representative
-> of the performance characteristics of our hosted offering. This image bundles
-> several services into the same container, while in our hosted offering we
-> run these services scaled across many machines.
+The Materialize Emulator is an all-in-one Docker image available on Docker Hub, offering the fastest way to get hands-on experience with Materialize in a local environment.
+
+|                           | **Materialize Emulator**                                          | **Materialize**                                                              |
+|---------------------------|-------------------------------------------------------------------|------------------------------------------------------------------------------|
+| **Description**           | The fastest option for prototyping or internal testing with Materialize.              | Materialize's cloud hosted SaaS solution for critical operational workloads. [Try it free](https://materialize.com/try/) or [book a demo](https://materialize.com/demo/)            |
```

Comment:
Although the ../try link redirects, should we just update it to the /register page?

Reply:
yes, good catch ✅